mods/ctf_pvp_engine/ctf_protect/init.lua

```diff
@@ -3,7 +3,7 @@ ctf.register_on_init(function()
 	ctf.log("chat", "Initialising...")
 
 	-- Settings: Chat
-	ctf._set("node_ownership",          true)
+	ctf._set("node_ownership",			true)
```

Comment:
```suggestion
	ctf._set("node_ownership", true)
```
^ since there's no need for alignment or indentation here.